react-components/src/components/RevealToolbar/SelectSceneButton.tsx

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type ReactElement } from 'react';
+
+import { Button, Dropdown, Menu, Tooltip as CogsTooltip } from '@cognite/cogs.js';
+import { type DmsUniqueIdentifier, use3dScenes } from '../..';
```

Comment:
change it to proper path like `../../hooks/use3dScenes`

Reply:
Fixed in [f2e5137](https://github.com/cognitedata/reveal/pull/4051/commits/f2e51372ec8afba5d2ed7773212bbbbd6b2f27cf)